fixture/failed.test.js

```diff
@@ -0,0 +1,3 @@
+test('1 is not equal 2', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/codingstyle-cn/hlj/pullRequest?prid=7143429)
```suggestion
test("1 is not equal 2", () => {
```